collect_app/src/main/java/org/odk/collect/android/utilities/WidgetAppearanceUtils.java

```diff
@@ -103,6 +102,11 @@ public static String getSanitizedAppearanceHint(FormEntryPrompt fep) {
         return appearance;
     }
 
+    /** Returns whether the specified appearance constant (defined above) is present. */
```

Comment:
It's meant to refer to all the string constants that are defined in this file—the constants are all appearance values, and the second argument to this function should be one of these constants.  I slightly reworded this, hoping it will help.

        /** Returns whether an appearance is present.  (Appearances are the constants above.) */

Reply:
🤦🏻‍♀️ Yes, that's great. I probably could have figured that out.